ice40/regressions/Makefile

```diff
@@ -0,0 +1,8 @@
+NPNR = ../../nextpnr-ice40
+JSON = $(wildcard */*.json)
+OUTPUT = $(patsubst %.json,%.log,$(JSON))
+
+all: $(NPNR) $(OUTPUT)
+
+%.log: %.json $(wildcard %.pcf)
```

Comment:
This doesn't do what it's supposed to - on my machine (Gentoo, GNU Make 4.2.1), it ends up not calling nextpnr with `--pcf` for the submitted test case.

Reply:
Thanks! I spotted some flaws when I was building more testcases.

Perhaps this is better for you?
```
%.log: %.json $(wildcard %.pcf) $(wildcard %.npnr) $(NPNR)
        $(NPNR) --json $(word 1,$^) $(if $(wildcard $*.pcf),--pcf $*.pcf,)) > $@ 2>&1
```
I'm not entirely sure why it would fail on your make...